lib/elixir/lib/application.ex

```diff
@@ -334,6 +334,22 @@ defmodule Application do
   end
 
   @doc """
+  Returns a list with information about the applications which are currently running.
+  """
+  @spec started_applications :: [tuple]
+  def started_applications do
```

Comment:
I think it should be `started_applications/1` with default timeout value and call `:application.which_applications/1`.